src/rad/inspector/wxfbadvprops.cpp

```diff
@@ -606,7 +645,7 @@ wxVariant wxFBBitmapProperty::ChildChanged(wxVariant& thisValue, const int child
             break;
         }
 
-        // file_path || id || resource_name
+        // file_path || id || resource_name || xrc_name
```

Comment:
What does this comment mean?

Reply:
This comment shows the name of the value the child at this index represents for the source types that use it (all source types currently use this child index). Hmm, this sounds quite complicated, this is the name of the row in the property grid below the row source 😄.